darwin/exporter/formats/mask.py

```diff
@@ -10,7 +10,7 @@
 try:
     from numpy.typing import NDArray
 except ImportError:
-    NDArray = Any  # type:ignore
+    NDArray = Any  # type:ignore # noqa F821
```

Comment:
We need this `NDArray = Any` to support darwin-py working with Databricks, which relies on a very old `NumPy` version